exercises/E-conditionals/exercise.js

```diff
@@ -6,8 +6,16 @@
   If Daniel is a student, print out "Hi, I'm Daniel, I'm a student."
 */
 
-var name = "Daniel";
-var danielsRole = "mentor";
+const name = "Daniel";
+const danielsRole = "mentor";
+
+if (danielsRole === "mentor") {
+  const message = `Hi, I'm ${name}, I'm a mentor`;
+  console.log(message);
+} else if (danielsRole === "student") {
+  const message = `Hi, I'm ${name}, I'm a student`;
+  console.log(message);
+}
```

Comment:
If you move the `console.log` after the `if`/`else` statement, you only need to use it once. I.e.

```js
if (danielsRole === "mentor") {
  const message = `Hi, I'm ${name}, I'm a mentor`;
} else if (danielsRole === "student") {
  const message = `Hi, I'm ${name}, I'm a student`;
}

console.log(message);
```